04_MMU/error-codes.yaml

```diff
@@ -338,24 +376,28 @@ Errors:
   action: [ResetMMU,DisableMMU]
   id: "FW_UPDATE_NEEDED"
   approved: true
+  type: USER_ACTION
 
 - code: "04505"
   title: "FW RUNTIME ERROR"
   text: "Internal runtime error. Try resetting the MMU or updating the firmware."
   action: [ResetMMU]
   id: "FW_RUNTIME_ERROR"
   approved: true
+  type: ERROR
 
 - code: "04506"
   title: "UNLOAD MANUALLY"
   text: "Filament detected unexpectedly. Ensure no filament is loaded. Check the sensors and wiring."
   action: [Retry]
   id: "UNLOAD_MANUALLY"
   approved: true
+  type: USER_ACTION
 
 - code: "04507"
   title: "FILAMENT EJECTED"
   text: "Remove the ejected filament from the front of the MMU."
+  text2: "Tip: Filament is automatically ejected when the end of the spool is reached."
```

Comment:
On MK3S we don't have the option to show a second text at the moment.
Does the text fits on the MK4/3.9/3.5  LCD screen?
The second text would be split in multiple screens on MK3/S/+
```
 |01234567890123456789
1|Tip: Filament is
2|automatically
3|ejected when the end
4|of the spool is

1|reached.
2|
3|
4|>Continue          w
```